routers/stops.py

```diff
@@ -69,15 +89,14 @@ def list_stops(current_user: User = Depends(get_current_user)):
         # Return an empty list if no stops are found
         return []
 
-#TODO does not depend on the user
+#TODO doesn't need to be an endpoint, only used inside create_stop
 @router.get("/stops_sorted")
-def get_stops_sorted(current_user: User = Depends(get_current_user), lat: float = 0, long: float = 0):
```

Comment:
I'm not 100% sure, but it's possible that by removing the user parameter completely we'll allow unauthorised access to the endpoint. Do we want this?